src/reanimated2/RegistersState.js

```diff
@@ -0,0 +1,40 @@
+import NativeModule from './NativeReanimated';
+
+
+export default class RegistersState {
```

Comment:
Can you export instance instead of class?

Reply:
fixed d84f2ab0bb3275696ef078668869069e14f1f7b9